api/src/main/java/com/code4ro/nextdoor/security/jwt/JwtTokenProvider.java

```diff
@@ -60,6 +84,7 @@ public boolean validateToken(final String authToken) {
         } catch (IllegalArgumentException ex) {
             LOGGER.error("JWT claims string is empty.");
```

Comment:
it's a best practice to pass exception to the logger to have it's stacktrace, event if you have a custom logging message the stacktrace provides valuable info